kernel/src/vm/vmar/vm_mapping.rs

```diff
@@ -435,19 +400,19 @@ impl VmMapping {
 /// with any child VMARs. And unless specified otherwise, it is not allowed
 /// to overlap with any existing mapping, either.
 pub struct VmarMapOptions<R1, R2> {
-    parent: Vmar<R1>,
-    vmo: Option<Vmo<R2>>,
-    perms: VmPerms,
-    vmo_offset: usize,
-    vmo_limit: usize,
-    size: usize,
-    offset: Option<usize>,
-    align: usize,
-    can_overwrite: bool,
+    pub(in crate::vm::vmar) parent: Option<Vmar<R1>>,
```

Comment:
`VmarMapOptions` can be moved to `vmar/mod.rs` to avoid these `pub(in crate::vm::vmar)`.